__tests__/server.test.js

```diff
@@ -24,13 +24,112 @@ describe('Person Route', () => {
 
 //REST route tests
 
-//Food route
-describe('REST route tests', () => {
+describe('Food route tests', () => {
   beforeEach(() => sequelize.sync());
 
   test('Gets all food items', async() => {
     const response = await request.get('/food');
     expect(response.statusCode).toBe(200);
+    const food = response.body[0];
+    expect(food.foodType).toEqual('noodles');
+  });
+
+  test('Gets one food item', async() => {
+    const response = await request.get('/food/4');
+    expect(response.statusCode).toBe(200);
+    const food = response.body;
+    expect(food.foodType).toEqual('yakitori');
+    expect(food.quantity).toEqual(7);
+  });
+
+  test('Creates a new food item', async() => {
+    const response = await request.post('/food').send({
+      foodType: 'french fries',
+      quantity: 3,
+     });
+    expect(response.statusCode).toBe(200);
+    const id = response.body.id;
+    const food = await request.get(`/food/${id}`);
+    expect(food.statusCode).toBe(200);
+    expect(food.body.foodType).toEqual('french fries');
+  });
+
+  test('Creates a new food item with sauces', async() => {
+    const response = await request.post('/food').send({
+      foodType: 'french fries',
+      quantity: 3,
+      sauceType: 'ketchup',
+     });
+    expect(response.statusCode).toBe(200);
+    const id = response.body.id;
+    const food = await request.get(`/food/${id}`);
+    expect(food.statusCode).toBe(200);
+    expect(food.body.foodType).toEqual('french fries');
+    expect(food.body.sauceType).toEquel('ketchup');
+  });
+
+  test('Updates an existing food item', async() => {
+    const response = await request.put('/food/2').send({
+      foodType: 'soba',
+    });
+    expect(response.statusCode).toBe(200);
+    const food = response.body;
+    expect(food.foodType).toEqual('soba');
+    expect(food.quantity).toEqual(4);
+  });
+
+  test('Deletes an existing food item', async() => {
+    await request.delete('food/2');
+    const response = await request.get('/food');
+    expect(response.body[1].foodType).toEqual('noodles');
   });
   
-})
\ No newline at end of file
+});
+
+describe('Drink route tests', () => {
```

Comment:
Test the post routes first so that way you can set up a 'mock' database and use it to test your other REST methods for the drink route